core/blockchain_snapshot_test.go

```diff
@@ -570,7 +574,7 @@ func TestHighCommitCrashWithNewSnapshot(t *testing.T) {
 	for _, scheme := range []string{rawdb.HashScheme, rawdb.PathScheme} {
 		expHead := uint64(0)
 		if scheme == rawdb.PathScheme {
-			expHead = uint64(4)
+			expHead = uint64(6)
```

Comment:
Same here, would be good to have a comment that in path mode we expect the database to be consistent up to block 6, since that was the last comitted